bonelab/cli/RapidPrototypeLetters.py

```diff
@@ -0,0 +1,487 @@
+          '{:8s} = {:8.2f} mm'.format('Bounds 2',cube.GetOutput().GetBounds()[2]),
+          '{:8s} = {:8.2f} mm'.format('Bounds 3',cube.GetOutput().GetBounds()[3]),
+          '{:8s} = {:8.2f} mm'.format('Bounds 4',cube.GetOutput().GetBounds()[4]),
+          '{:8s} = {:8.2f} mm'.format('Bounds 5',cube.GetOutput().GetBounds()[5]))
+  
+  radius = 1.0
+  post = vtk.vtkCylinderSource()
+  post.SetHeight(9.0)
+  post.SetRadius(radius)
+  post.SetResolution(20)
+  post.CappingOn()
+  
+  transform = vtk.vtkTransform()
+  transform.RotateX(90.0)
+
+  transformPost = vtk.vtkTransformFilter()
+  transformPost.SetInputConnection( post.GetOutputPort() )
+  transformPost.SetTransform( transform )
+  transformPost.Update()
+
+  post_center = transformPost.GetOutput().GetCenter()
+  message('Post center:',
+          '{:8s} = {:8.2f} mm'.format('X',post_center[0]),
+          '{:8s} = {:8.2f} mm'.format('Y',post_center[1]),
+          '{:8s} = {:8.2f} mm'.format('Z',post_center[2]))
+
+  transform1 = vtk.vtkTransform()
+  transform1.Translate(cube.GetOutput().GetBounds()[0]+(radius*2), cube.GetOutput().GetBounds()[2]+(radius*2), 2)
+
+  transform2 = vtk.vtkTransform()
+  transform2.Translate(cube.GetOutput().GetBounds()[0]+(radius*2), cube.GetOutput().GetBounds()[3]-(radius*2), 2)
+  
+  transform3 = vtk.vtkTransform()
+  transform3.Translate(cube.GetOutput().GetBounds()[1]-(radius*2), cube.GetOutput().GetBounds()[2]+(radius*2), 2)
+  
+  transform4 = vtk.vtkTransform()
+  transform4.Translate(cube.GetOutput().GetBounds()[1]-(radius*2), cube.GetOutput().GetBounds()[3]-(radius*2), 2)
+  
+  transformPost1 = vtk.vtkTransformFilter()
+  transformPost1.SetInputConnection( transformPost.GetOutputPort() )
+  transformPost1.SetTransform( transform1 )
+  transformPost1.Update()
+  
+  transformPost2 = vtk.vtkTransformFilter()
+  transformPost2.SetInputConnection( transformPost.GetOutputPort() )
+  transformPost2.SetTransform( transform2 )
+  transformPost2.Update()
+  
+  transformPost3 = vtk.vtkTransformFilter()
+  transformPost3.SetInputConnection( transformPost.GetOutputPort() )
+  transformPost3.SetTransform( transform3 )
+  transformPost3.Update()
+  
+  transformPost4 = vtk.vtkTransformFilter()
+  transformPost4.SetInputConnection( transformPost.GetOutputPort() )
+  transformPost4.SetTransform( transform4 )
+  transformPost4.Update()
+  
+  post12 = JoinPolyData(transformPost1.GetOutput(),transformPost2.GetOutput())
+  post123 = JoinPolyData(post12.GetOutput(),transformPost3.GetOutput())
+  post1234 = JoinPolyData(post123.GetOutput(),transformPost4.GetOutput())
+  post1234.Update()
+  
+  cube_and_posts = JoinPolyData(cube.GetOutput(),post1234.GetOutput())
+  
+  mat4x4 = visualize_actors( cube_and_posts.GetOutputPort(), bone1234.GetOutputPort()  )
+  #mat4x4 = visualize_actors( cube_and_posts.GetOutputPort(), None  )
+  
+  #ofile = "/Users/skboyd/Documents/projects/FormLabs/radius_time_series/bone_mask_post_and_table.stl" 
```

Comment:
You could remove the path or remove all the comments.
```suggestion
  #ofile = "bone_mask_post_and_table.stl" 
```